devices/dimmer_switch.py

```diff
@@ -0,0 +1,16 @@
+import Domoticz
+from devices.device import Device
+
+
+class DimmerSwitch(Device):
```

Comment:
I guess you could just reuse DimmerSwitch from `devices.switch.dimmer_switch.py`